src/server/server.js

```diff
@@ -171,10 +173,17 @@ module.exports = class DvlpServer {
   start() {
     return new Promise((resolve, reject) => {
       // Force reject if never started
-      const timeoutID = setTimeout(() => {
-        debug('server not started after timeout');
-        reject(Error('unable to start server'));
-      }, START_TIMEOUT_DURATION);
+      const timeoutIntervalID = setInterval(() => {
+        // Needed as intercepting and watching files negatively affects startup time
+        const didTimeout =
+          Date.now() - this.lastInterceptFileRead > START_TIMEOUT_DURATION;
+
+        if (didTimeout) {
+          clearInterval(timeoutIntervalID);
+          debug('server not started after timeout');
+          reject(Error('unable to start server'));
```

Comment:
Perhaps we should take the opportunity to update the Error message: 
```suggestion
          reject(Error(`unable to start server: application server did not respond within the ${START_TIMEOUT_DURATION}ms startup window`));
```